CONTRIBUTING.md

```diff
@@ -1,22 +1,56 @@
-A number of developers met in person and [sprinted on specs](https://github.com/radiantearth/boulder-sprint/)
-and made a number of key decisions. 
-
-The best way to contribute is to try to implement the specification and give feedback on what worked
-well and what could be improved. Currently there are a few major things to still work out, so if you
-are interested in implementing you should jump on our [gitter channel](https://gitter.im/SpatioTemporal-Asset-Catalog/Lobby)
-and check in on how ready the spec is and how you can support it. For the next bit of time you'll
-probably be encouraged to just do what feels best in some areas, so we can capture best practices
-and standardize.
+Pull Requests are the primary method of contributing to the spec itself, and everyone is welcome to submit 
+changes. Suggestions for changes to the core will be taken with higher priority if a clear implementation 
+of STAC has been built that can highlight the problem. If the changes can be done as an [extension](extensions/) 
+instead of modifying the core files then that route is recommended first, and once there is uptake for the 
+extension it will be considered for core.
+
+We consider everyone using the specification to catalog their data to be a 'contributor', as STAC is
+really about the end result of more interoperable data, not just creating a spec for the sake of it.
+So if you want to help then the best thing you can do is make new catalogs or build software that uses
+the spec. And please do give us feedback. The best place to do so is on our 
+[gitter channel](https://gitter.im/SpatioTemporal-Asset-Catalog/Lobby).
+
+### Submitting Pull Requests
 
 Any proposed changes to the specification should be done as pull requests. Please make these
 requests against the [dev](https://github.com/radiantearth/stac-spec/tree/dev) branch (this will
-require you to switch from the default of 'master', which we keep so it displays first). Ideally a
-proposed change would also update all of the examples, but for now that is not required - the team
-can validate and update all examples before a release. But it is recommended to update at least a
-couple examples to reflect the change.
+require you to switch from the default of 'master', which we keep so it displays first). 
+
+Creating a Pull Request will show our PR template, which includes checkbox reminders for a number
+of things.
+
+* Adding an entry the [CHANGELOG](CHANGELOG.md). If the change is more editorial and minor then this 
+is not required, but any change to the actual specification should definitely have one.
+* Base the PR against dev, as mentioned above - even if the branch was made off of dev this reminds
+you to change the base in GitHub's PR creation page.
+* Make a ticket in the STAC API repo if anything here affects there.
+* Make sure the PR makes no breaking changes to the specification.
```

Comment:
That sounds like you are not allowed to do breaking changes at all, although we still do.

Reply:
Yeah, I got a little confused on that one. We are close to 1.0, and at that time we won't allow breaking changes. So went back and forth if it should be aspirational. But I'll tune it down for now.